speech_assets/utterances-for-generator.txt

```diff
@@ -6,41 +6,50 @@
 # Tool to expand the single line into the usable list of utterances
 # http://www.makermusings.com/amazon-echo-utterance-expander/
 
+# Check for duplicates after generating
 
-PlayPause (play/pause/resume)
 
-AMAZON.StopIntent (stop/cancel/shut up)
+PlayPause (play/pause/resume) (/song/video/episode/track/movie/playback)
 
-Skip (skip/next) (/song)
-Skip (listen to/play) next (/song)
 
-Prev (/listen to/play) previous (/song)
+Skip skip (/this) (/one/item/song/video/episode/track/movie)
 
-StartOver (replay/start over)
+Skip (listen to (/the) /play (/the)) next (song/track)
+Skip (watch to (/the) /play (/the)) next (video/episode/movie)
 
-Home navigate home
+Skip (listen to/play/watch) (/the) next (one/item) 
 
-Back (/navigate/go) back
-Up (/navigate/go) up
-Down (/navigate/go) down
-Right (/navigate/go) right
-Left (/navigate/go) left
-PageUp (/navigate/go) page up
-PageDown (/navigate/go) page down
+Prev (listen to/play) (/the/that) (previous/last) (song/track) (/again)
```

Comment:
Ah, good catch.  "next/previous" audio makes more sense in the context of the audio stream, so I'd leave it as it is.